tests/test_project/tests/test_admin.py

```diff
@@ -30,3 +31,10 @@ def test_radiusaccounting_changelist(self):
         url = reverse('admin:test_project_radiusaccounting_changelist')
         response = self.client.get(url)
         self.assertNotContains(response, 'Add accounting')
+
+    def test_alwayshaschangedmixin(self):
+        self.assertEqual(self.project_model.objects.all().count(), 0)
+        options = dict(name='test')
+        params = self._get_defaults(options)
+        path = reverse('admin:test_project_project_add')
+        obj = self.client.post(path, params)
```

Comment:
you need to test an object was created correctly here

Reply:
@nemesisdesign Do you know method how to show all parameters `formset` (I was trying to use `OperatorInline.formset.management_form` but it was returning `<django.utils.functional.cached_property object at 0x7f920c04be10>`) like `operator-TOTAL_FORMS`, `operator-INITIAL_FORMS` because I am getting error `django.core.exceptions.ValidationError: ['ManagementForm data is missing or has been tampered with']`

Reply:
this is enough:

```python
self.assertEqual(self.project_model.objects.count(), 1)
project = self.project_model.objects.first()
self.assertEqual(project.name, params['name'])
```